picard/coverart/providers/caa.py

```diff
@@ -26,8 +26,10 @@
     OrderedDict,
     namedtuple,
 )
+from functools import partial
 
 from PyQt5 import (
+    Qt,
```

Comment:
This is prolly the cause of the appveyor issue, see https://stackoverflow.com/questions/48514027/spyder3-python-ide-does-not-start-this-windows-version-does-not-support-the-re

In fact, it is incorrect, and this import should be removed, because `QListWidgetItem` belongs to `QtWidgets`

So replace:

```python
 item = Qt.QListWidgetItem(title)
```

with:

```python
 item = QtWidgets.QListWidgetItem(title)
```

And let's see if it fixes the issue.